.github/workflows/cron-staging.yml

```diff
@@ -44,4 +44,35 @@ jobs:
         if: runner.os == 'macOS'
         env:
           TEST_TIMEOUT: 120
-          OMP_NUM_THREADS: 1
\ No newline at end of file
+          OMP_NUM_THREADS: 1
+  docs:
+    name: docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - name: Pip cache
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-docs-${{ hashFiles('setup.py','requirements.txt','requirements-extras.txt','requirements-dev.txt','constraints.txt') }}
+      - name: Install Deps
+        run: |
+          python -m pip install -U tox
+          sudo apt-get install -y pandoc graphviz
+      - name: Build Docs
+        run: tox -edocs-parallel
```

Comment:
This will catch a problem the day after a new release (of anything) breaks something. Is that good enough or should run against terra main like we do with the tests so we can catch problems before release?

Reply:
My bad, thanks for catching. I've added a `docs-terra-main` environment now. `tox.ini` keeps getting more bloated, I hope can be addressed by switching to another tool.

Reply:
Yes, it is hard to prioritize everything. We could review nox and hatch again. One option might be to stick with tox and to use a separate task running tool like make or [just](https://github.com/casey/just) to define commands to run. I think we only need two tox environments (terra release and terra main) and its just different commands that we need to run. With `tox exec`, we can run different commands in the same tox environment rather than defining a separate environment for each command and using `tox run`.